cdptools/bin/clone_db.py

```diff
@@ -40,7 +44,15 @@ def pass_through(row, target_db, table):
 
 def main():
     args = Args()
-    source_db = c_db.CloudFirestoreDatabase(project_id=args.project_id)
+     
+    source_db = None
+    if args.source_project_id:
+        source_db = c_db.CloudFirestoreDatabase(project_id=args.source_project_id)
+        log.info("Cloning database with project_id={}".format(args.source_project_id))
+    else:
+        source_db = c_db.CloudFirestoreDatabase(credentials_path=args.source_credentials)
+        log.info("Cloning database with credentials={}".format(args.source_credentials))
+
```

Comment:
In the case that there is no default value of `cdp-seattle`, you will need to catch and raise when no DB can be created without either `source_project_id` or `source_credentials`.

Reply:
Nevermind solved by the parser group handling.